compass/LayersCalculator.java

```diff
@@ -77,8 +77,11 @@ public static void main(String[] args) throws IOException {
   public void run() {
     Path layersPath = Paths.get(config.layersPath);
     try {
-      Files.createDirectory(layersPath);
+      if (Files.notExists(layersPath)) {
+        Files.createDirectory(layersPath);
+      }
```

Comment:
From the javadocs:
```
Creates a new directory. The **check for the existence of the file** and the creation of the directory if it does not exist are a single operation that is atomic with respect to all other filesystem activities that might affect the directory
```

I am fine with being maybe more explicit so that the maintainer won't have to read javadocs. No harm being done here. Just wondering though, IDEA warned you about that?

Reply:
no, but it may throw `FileAlreadyExistsException`:
[createDirectory](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#createDirectory-java.nio.file.Path-java.nio.file.attribute.FileAttribute...-)
so I added it.

Reply:
ahh :blush: 
I should learn to read the javadocs